pytorch_lightning/utilities/logger.py

```diff
@@ -0,0 +1,130 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for loggers."""
+
+from argparse import Namespace
+from typing import Any, Dict, Generator, List, MutableMapping, Optional, Union
+
+import numpy as np
+import torch
+
+
+def _convert_params(params: Union[Dict[str, Any], Namespace]) -> Dict[str, Any]:
```

Comment:
please add unit tests for these functions outside of the integration tests with the Loggers

Reply:
Unit tests have been added! I moved the wandb unit test to this new file as well, as @akihironitta suggests here https://github.com/PyTorchLightning/pytorch-lightning/pull/11484#discussion_r789869762